lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -86,9 +90,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
           'crop_rotate' => $values['crop_rotate'],
         ]),
   ];
-
-  $response = drupal_http_request($client['base_url'] . '/reportbacks', $options);
-
+  $response = post($client['base_url'] . '/reportbacks', $options);
```

Comment:
To coincide better with what we're doing in **dosomething_northstar.module** mind renaming this function `_dosomething_rogue_build_http_client()` to `dosomething_rogue_client()` and ignore the stupid underscore `_` prefixing that's a dumb Drupal standard 😉